oauth2_http/java/com/google/auth/oauth2/JwtCredentials.java

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright 2019, Google Inc. All rights reserved.
```

Comment:
Google LLC.
No "All rights reserved"

Reply:
Our maven-checkstyle-plugins's java.header file validates against this format and has for years. We can change it, but it's outside the scope of this PR.

Reply:
That's probably out of date then. Current guidance and explicit text to copy is here:

https://g3doc.corp.google.com/company/teams/opensource/releasing/preparing.md?cl=head#license-headers

Reply:
Also note that this library was released under BSD 3 Clause License vs. our standard Apache 2.0

Reply:
Fixed